config/set/contentrepository-90.php

```diff
@@ -376,8 +375,12 @@
 
     /**
      * Neos.Neos:PrimaryContent
+     * Neos.Fusion:Attributes
      */
-    $rectorConfig->rule(FusionPrimaryContentRector::class);
+    $rectorConfig->ruleWithConfiguration(FusionPrototypeNameAddComment::class, [
```

Comment:
@kdambekalns Yeah, sorry. AFAIS there is wrong class used:
This needs to be: `FusionPrototypeNameAddCommentRector::class`

Reply:
It's pretty sh**ty, that the tests are not covering the "real" configuration.

Reply:
Yep, creating a PR right now…